crates/forge/tests/cli/config.rs

```diff
@@ -856,3 +857,348 @@ contract MyScript is BaseScript {
     pretty_err(&lib_toml_file, fs::write(&lib_toml_file, lib_config.to_string_pretty().unwrap()));
     cmd.forge_fuse().args(["build"]).assert_success();
 });
+
+#[cfg(not(feature = "isolate-by-default"))]
+forgetest_init!(test_default_config, |prj, cmd| {
+    cmd.forge_fuse().args(["config"]).assert_success().stdout_eq(str![
+        r#"
+[profile.default]
```

Comment:
These all are needed for revive?

Reply:
These are part of the standard config with revive configs included